src/features/content-explorer/content-explorer/__tests__/ContentExplorerHeaderActions.test.js

```diff
@@ -40,6 +40,17 @@ describe('features/content-explorer/content-explorer/ContentExplorerHeaderAction
             expect(wrapper.find('ContentExplorerBreadcrumbs').length).toBe(1);
         });
 
+        test('should render ContentExplorerFolderTreeBreadcrumbs when hasFolderTreeBreadcrumbs is true', () => {
+            const wrapper = renderComponent();
+            wrapper.setProps({ hasFolderTreeBreadcrumbs: true });
+
+            expect(wrapper.hasClass('content-explorer-header-actions')).toBe(true);
+            expect(wrapper.find('ContentExplorerSearch').length).toBe(1);
+            expect(wrapper.find('ContentExplorerNewFolderButton').length).toBe(1);
```

Comment:
since these are being tested in the above test, you don't need to check them in this one. this allows this test to have a smaller focus